.gitignore

```diff
@@ -138,3 +138,9 @@ dask-worker-space/
 
 # tmp directory
 tmp/
+
```

Comment:
Also ignore `_results` and `_html` (though that should get fixed in the template eventually - https://github.com/lincc-frameworks/python-project-template/issues/269)

Reply:
got it